tests/e2e/envfuncsext/nebulacluster.go

```diff
@@ -165,6 +166,16 @@ func WaitNebulaClusterReady(opts ...NebulaClusterOption) env.Func {
 				"generation", nc.Generation,
 			)
 
+			if nc.Status.Graphd.Phase == appsv1alpha1.RunningPhase && nc.Status.Metad.Phase == appsv1alpha1.RunningPhase && nc.Status.Storaged.Phase == appsv1alpha1.RunningPhase {
+				klog.V(4).Infof("First running phase detected for nc cluster. Trying again after 5 sec to see if nc cluster is really running")
+				time.Sleep(5 * time.Second)
+
+				if err = cfg.Client().Resources().Get(ctx, nc.GetName(), nc.GetNamespace(), nc); err != nil {
+					klog.ErrorS(err, "Get NebulaCluster failed", "namespace", nc.Namespace, "name", nc.Name)
+					return false, err
+				}
+			}
+
```

Comment:
@veezhang This is to make sure all components are really in running after scaling a cluster before running the next this. Removing this results in the following error, because Storaged is still in ScaleOut phase when the next scale test starts but goes into running mode after a second. The sleep is actually not needed was put there initially as a delay so it didn't check again immediately, removed now.
```
Error: UPGRADE FAILED: cannot patch "e2e-nc-0-1648507714a627f9dec9b46" with kind NebulaCluster: admission webhook "nebulaclustervalidating.nebula-graph.io" denied the request: [spec.storaged: Forbidden: field is immutable while in ScaleOut phase, spec.storaged.replicas: Invalid value: 4: field is immutable while not in Running phase]: exit status 1
```

Reply:
Unable to reproduce in the latest operator version. Considering this fixed. This code has been removed.